src/threads/table.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { dynamodb, toDBItem } from '../dynamodb';
+import { ForumTable } from './table';
+
+const putItem = (threadId: string, timestamp: number, data: Record<string, unknown>) => dynamodb.putItem({
```

Comment:
A "loadFixture" function receiving an array would probably be more clear (in terms of testing) / practical, but it can be done later.